AdminApi.Tests.Integration/Models/TierQuotaDefinitionDTO.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace AdminApi.Tests.Integration.Models;
+
+public class TierQuotaDefinitionDTO
+{
+    public string Id { get; set; }
+    public MetricDTO Metric { get; set; }
+    public int Max { get; set; }
+    public string Period { get; set; }
```

Comment:
The mapping of the JSON response to a `QuotaPeriod` object fails because the AdminAPI is returning a string (due to the usage of a converter). Should we implement another converter of sorts on the Tests?